src/App/AppContainer.js

```diff
@@ -1,149 +1,172 @@
+// import React/Redux dependencies
 import React, { Component } from 'react';
-import {Route, Redirect} from 'react-router';
+import { Route } from 'react-router';
 import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { loadResources } from '../action/resourceDataAction';
+import { getAllSites } from '../api/directoryGoogleSheets';
+
+// import components
+import { Badge } from 'reactstrap';
+import Loader from 'react-loader-spinner';
 import Header from '../components/Header/Header';
 import MapPage from '../components/MapPage/MapPage';
 import AdminPage from '../components/AdminPage/AdminPage';
 import { SplitScreenTogglePane } from '../components/AdminPage/SplitScreenTogglePane';
 import SavedResourcePanel from '../components/SavedResources/SavedResourcePanel';
-import { loadResources } from '../action/resourceDataAction';
-import {getAllSites} from '../api/directoryGoogleSheets';
-import { Badge } from 'reactstrap';
-import styles from './App.module.css';
-import Loader from 'react-loader-spinner'
-import PropTypes from 'prop-types'
-import NotFoundPage from '../components/NotFoundPage/NotFoundPage'
-
-const envSheetId = process.env.REACT_APP_GOOGLE_SHEETS_ID
-const revereSheetId = '1QolGVE4wVWSKdiWeMaprQGVI6MsjuLZXM5XQ6mTtONA';
-
-function sheetIdFromPath(directory, path){
-    for (var i=0; i < directory.length; i++) {
-        if (directory[i].path === path) {
-            return directory[i].sheetId;
-        }
+import NotFoundPage from '../components/NotFoundPage/NotFoundPage';
+
+const envSheetId = process.env.REACT_APP_GOOGLE_SHEETS_ID;
+
+// unused code
+// const revereSheetId = '1QolGVE4wVWSKdiWeMaprQGVI6MsjuLZXM5XQ6mTtONA';
```

Comment:
Thanks for noting this and leaving it here for now - we'll evaluate whether this is needed.